release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1443,7 +1440,20 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
 
 *Web console (Developer perspective)*
 
+* This update adds an error message to the *Workloads* menu of the monitoring dashboard if a user does not have access privileges. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930546[*BZ#1930546*])
+
+* This update fixes the sort order and color of `ImageManifestVuln` objects for the Quay Security Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[*BZ#1942716*])
 
+* Previously, the *Topology* view in the Developer perspective did not load if OpenShift namespace templates were not available. This update fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[*BZ#1949810*])
+
+* Previously, when you created a quick start tutorial in the web console and specified prerequisites, the prerequisites were displayed as a paragraph instead of a list. Lists are now displayed correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905147[*BZ#1905147*])
+
+* Previously, after importing a devfile, the container did not start successfully. This occurred due to the invalid inclusion of `buildguidanceimage-placeholder` into the deployment. When only the correct container image is included, that image opens as expected. With the latest version, a placeholder image could be used as well. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[*BZ#1952214*])
```

Comment:
s/could be used as well/could also be used  IBM style guide says not to use _as well_ to mean _and_.